### PR TITLE
DEV 946 rss feed of bounties

### PR DESCRIPTION
**Pull request by vhcsilva**

### CODE REVIEW

File: templates/rss.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+export const rssTemplate = `<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (moshmage) [inline templates/rss.ts:1]: wouldn't it make sense for this file to be `tsx` ?

File: db/migrations/20230113214456-add-rss-time-to-settings.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    await Settings.create({
+      key: "rssTtl",
+      value: "1",
+      visibility: "public",
+      type: "string"
+    });
+  },
+
+  async down (queryInterface, Sequelize) {
+
```
Comment (moshmage) [inline db/migrations/20230113214456-add-rss-time-to-settings.js:18]: remove the changes made on `up`

File: templates/rss.ts (hunk 1 above)
Comment (vhcsilva) [reply templates/rss.ts:1]: @moshmage as this file only exports a string and not HTML, i think we can use the `.ts` extension

Comment (moshmage) [reply templates/rss.ts:1]: if this is a template for handlebars, use `hbs` instead; If this is "html with javascript" use `tsx`

**Review by moshmage — APPROVED** [review]

**Pull request merged**